test/ex_doc/formatter/html_test.exs

```diff
@@ -305,7 +305,7 @@ defmodule ExDoc.Formatter.HTMLTest do
                ~r{<h2 id="header-sample" class="section-heading">.*<a href="#header-sample" class="hover-link"><span class="icon-link" aria-hidden="true"></span></a>.*<code(\sclass="inline")?>Header</code> sample.*</h2>}ms
 
       assert content =~
-               ~r{<h2 id="more-than" class="section-heading">.*<a href="#more-than" class="hover-link"><span class="icon-link" aria-hidden="true"></span></a>.*more &gt; than.*</h2>}ms
```

Comment:
> `more &gt; than`

we had this before `f553cc0` which introduced the regression, I guess I added escaping not to break this test. I guess it's ok to remove escaping from this `h2` and from JSON as tested on line 338, I suppose we didn't need to escape them in the first place.

Reply:
@wojtekmach do you know what was the previous code that escaped this `>` character? I would like to have a look at it.

Reply:
It was probably within earmark.

Reply:
and what changed? Earmark or ExDoc?